FogCreek/Glitch-Community
# Sponge credit: add analytics events for my stuff button

## Links
* https://sponge-credit.glitch.me/

## Changes:
* adds analytics events every time someone clicks the bookmarks button

## How To Test:
* enable my stuff on the secret page
* click on the my stuff button on a user page, collection page, search page, team page, home page
* search in segment's debugger for the event, you should have enough context to know where the button was pressed

## Feedback I'm looking for:
- seem good to go?

## Things left to do before deploying:
- [x] gunna check in with ezra to see if I can get segment creds to verify stuff
